sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/ComputeNodeIntegrationTests.cs

```diff
@@ -829,7 +839,7 @@ public void ComputeNodeUploadLogs()
                         DateTime timeoutAt = DateTime.UtcNow.AddMinutes(2);
                         while (DateTime.UtcNow < timeoutAt)
                         {
-                            blobs = container.ListBlobs();
+                            blobs = await BlobStorageExtensions.ListBlobs(container);
```

Comment:
Does this exist because storage doesn't have continuation handling in their netcore SDK?

Reply:
Am not sure what you mean, but I did make use of the continuation token in the implementation of the extension method.[here](https://github.com/Azure/azure-sdk-for-net/pull/6362/files#diff-b2ca8962af117800d7407f8b9ac85c57)

Reply:
I mean that the Azure Storage SDK usually does this for you. The ListBlobs method we were calling previously already did this - I'm wondering why we can't just use an existing ListBlobs from the storage SDK itself rather than write our own extension

Reply:
The ListBlobs method is not available in .netcore2.1